aviary/modules/binning/scripts/finalise_recovery.py

```diff
@@ -0,0 +1,29 @@
+import os
+from pathlib import Path
+import glob
+
+if __name__ == '__main__':
+    final_bins = snakemake.input.final_bins
+    coverage_file = snakemake.input.coverm
+    contig_coverage = snakemake.input.contig_coverage
+    gtdbtk = snakemake.input.gtdbtk
+    singlem = snakemake.input.singlem
+
+    os.chdir('bins/')
+
+    if len(coverage_file) > 0:
+        os.symlink(f"../{coverage_file}", "./")
+    
+    if len(contig_coverage) > 0:
+        os.symlink(f"../{contig_coverage}", "./")
+    
+    os.chdir('..')
+    if len(gtdbtk) > 0:
+        os.symlink(f"{gtdbtk}", "taxonomy")
+    if len(singlem) > 0:
+        os.symlink(f"{singlem}", "diversity")
```

Comment:
Can these handle the symlink already existing? If it fails then they wont be removed by snakemake because they aren't outputs.

Reply:
https://docs.python.org/3/library/os.html#os.symlink The docs don't mention throwing an error, but yeah it does error if file exists. I'll fix